survey/5-tech.yml

```diff
@@ -332,4 +335,23 @@ questions:
       - Neo4j
       - Apache Cassandra
       - BigTable
+      - SQLite
+      - Other
+
+  - label: "Do you write tests for your code?(Don't be afraid to be honest)"
```

Comment:
I would rather rewrite this one to be more of a" what type of tests are you writing (unit/integration... ) with a none option and other. Fombine both into 1

Reply:
Yes

Reply:
done